lib/rules/no-mixed-operators.js

```diff
@@ -153,7 +168,7 @@ module.exports = {
          * @returns {Token} The operator token of the node.
          */
         function getOperatorToken(node) {
-            return sourceCode.getTokenAfter(node.left, astUtils.isNotClosingParenToken);
+            return sourceCode.getTokenAfter(node.left || node.test, astUtils.isNotClosingParenToken);
```

Comment:
Thoughts on abstracting `node.left || node.test` into a utility function since it's repeated a few times?

Reply:
Hmm. I get that it's repeated. I am worried if it will be an overkill to create a new function just for this. When we add more complexity to the same rule which brings in a lot of the same or related computations, I see a necessity to make it a function. Thoughts?

Reply:
Just to be clear, I meant a private utility function in this module that would document the operation happening here (since it seems to be a repeated calculation). Just a suggestion - I’ll defer to your judgment!

Reply:
I moved that logic to a function (y)